include/cripts/Epilogue.hpp

```diff
@@ -752,10 +752,12 @@ TSPluginInit(int argc, const char *argv[])
     inst->NeedCallback(enabled_txn_hooks);
     TSContDataSet(contp, context);
     TSHttpHookAdd(TS_HTTP_TXN_START_HOOK, contp); // This acts similarly to the DoRemap callback
+  } else if (needs_glb_init) {
```

Comment:
I don't remember much of this code, but would this not leak the context and inst now ?